quarkchain/cluster/shard_state.py

```diff
@@ -1995,4 +2015,23 @@ def get_total_balance(
         limit: int,
         starter: Optional[bytes] = None,
     ) -> Tuple[int, bytes]:
-        return 42, b""
+        """
+        starter should be exclusive
+        """
+        evm_state = self._get_evm_state_from_hash(block_hash)
+
+        trie = evm_state.trie.trie
+        key = trie.next(bytes(32) if not starter else starter)
```

Comment:
this seems wrong? starter should be an address, so the first key should be got from `evm_state.trie.db`?

Reply:
do you mean we map the starter addr to its key?

Reply:
yes, the I/O of the function both should be address